### ISSUE TITLE
reference_interfaces fails to build

### ISSUE THREAD

**Issue by nightduck** [ros-realtime/reference-system#95]
**Describe the bug**
Unable to build. Error message:

```
--- stderr: reference_interfaces                                                                                    
CMake Error at CMakeLists.txt:41 (add_custom_command):
  No TARGET 'reference_interfaces__python' has been created in this
  directory.
```

**To Reproduce**
1. Followed all instructions in [quick setup](https://github.com/ros-realtime/reference-system/blob/main/autoware_reference_system/README.md#quick-start) using latest verisons of this repo and ros2_tracing
2. Manually install some packages that rosdep misses: `sudo apt-get install ros-rolling-ament-mypy ros-rolling-ament-cmake-mypy ros-rolling-tracetools-analysis ros-rolling-ros-testing`
3. Build `colcon build --cmake-args -DRUN_BENCHMARK=TRUE -DTEST_PLATFORM=TRUE`

**Expected behavior**
Package should build

**Desktop (please complete the following information):**
 - OS: Ubuntu Focal

**Additional context**
Running in a docker container, with binaries of ROS2 rolling installed in it.

**Comment by christophebedard** [comment]
IIRC removing/commenting out those lines fixed this issue:

```diff
diff --git a/reference_interfaces/CMakeLists.txt b/reference_interfaces/CMakeLists.txt
index c8baf80..31ca5d4 100644
--- a/reference_interfaces/CMakeLists.txt
+++ b/reference_interfaces/CMakeLists.txt
@@ -33,14 +33,14 @@ rosidl_generate_interfaces(${PROJECT_NAME}
 
 ament_auto_package()
 
-# fix rosidl_generator_py bug #143
-# https://github.com/ros2/rosidl_python/issues/143
-set(GENERATED_FILE "${CMAKE_CURRENT_BINARY_DIR}/rosidl_generator_py/${PROJECT_NAME}/msg/_transmission_stats.py")
-
-message(STATUS "checking generated file: ${GENERATED_FILE}")
-add_custom_command(
-  TARGET ${PROJECT_NAME}__python
-  POST_BUILD
-  COMMAND sed -i "s/all(val >= 0 and val) < 256/all(ord(val) >= 0 and ord(val) < 256/" ${GENERATED_FILE}
-  COMMENT "Check generated IDL files for extra parenthesis..."
-  VERBATIM)
+# # fix rosidl_generator_py bug #143
+# # https://github.com/ros2/rosidl_python/issues/143
+# set(GENERATED_FILE "${CMAKE_CURRENT_BINARY_DIR}/rosidl_generator_py/${PROJECT_NAME}/msg/_transmission_stats.py")
+
+# message(STATUS "checking generated file: ${GENERATED_FILE}")
+# add_custom_command(
+#   TARGET ${PROJECT_NAME}__python
+#   POST_BUILD
+#   COMMAND sed -i "s/all(val >= 0 and val) < 256/all(ord(val) >= 0 and ord(val) < 256/" ${GENERATED_FILE}
+#   COMMENT "Check generated IDL files for extra parenthesis..."
+#   VERBATIM)
```

**Comment by nightduck** [comment]
Building with galactic also fixes it. This repo doesn't seem to support the rolling release.
It's important that you checkout the galactic branch of ros2_tracing as well.

Added a note in PR #97
